### ISSUE TITLE
Increasing node.js log level?

### ISSUE THREAD

**Issue by zenlord** [Requarks/wiki#1902]
**Question**
I am stuck in a situation where wiki.js fails to start, but the logs don't contain enough information, even if the log level in the wiki.js configuration file has been set to 'silly'.

This is the error output, no matter which log level I set in wiki.js:

```
May 16 13:51:25 X systemd[1]: Started Wiki.js.
May 16 13:51:26 X node[2826]: Loading configuration from /var/www/wiki.[DOMAIN]/wiki/config.yml... OK
May 16 13:51:26 X node[2826]: 2020-05-16T11:51:26.926Z [MASTER] info: =======================================
May 16 13:51:26 X node[2826]: 2020-05-16T11:51:26.929Z [MASTER] info: = Wiki.js 2.3.81 ======================
May 16 13:51:26 X node[2826]: 2020-05-16T11:51:26.929Z [MASTER] info: =======================================
May 16 13:51:26 X node[2826]: 2020-05-16T11:51:26.929Z [MASTER] info: Initializing...
May 16 13:51:27 X node[2826]: 2020-05-16T11:51:27.374Z [MASTER] info: Using database driver pg for postgres [ OK ]
May 16 13:51:27 X node[2826]: 2020-05-16T11:51:27.381Z [MASTER] info: Connecting to database...
May 16 13:51:27 X node[2826]: 2020-05-16T11:51:27.416Z [MASTER] info: Database Connection Successful [ OK ]
May 16 13:51:28 X node[2826]: 2020-05-16T11:51:28.193Z **[MASTER] error: Unexpected identifier**
May 16 13:51:28 X systemd[1]: wiki.service: Main process exited, code=exited, status=1/FAILURE
May 16 13:51:28 X systemd[1]: wiki.service: Failed with result 'exit-code'.
```

I guess this error is triggered by node before wiki.js is started - any pointers how to increase the log level  of node?

**Host Info (please complete the following information):**
OS: Debian Buster (with systemd service file)
Wiki.js version: 2.3.81
Database engine: postgres 11

**Comment by NGPixel** [comment]
What version of node are you using?

**Comment by zenlord** [comment]
Version 10.19 (from the Debian Buster repo)
https://packages.debian.org/buster/nodejs

Thx!

**Comment by czytom** [comment]
Same issue here with newest Wiki.js 2.4.107, silly logLevel looks like not applied

**Comment by jschirrmacher** [comment]
I can confirm that `logLevel` configuration is ignored with version 2.5.144 as well.

**Comment by kingfisher77** [comment]
Same here with latest Wiki.js

**Comment by kmpm** [comment]
Same as #2497 . Will apparently be fixed in 3.0
